src/components/NewClusterModal.js

```diff
@@ -405,6 +482,26 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           h(ButtonPrimary, { onClick: () => this.createCluster() }, ['REPLACE'])
         ])
       ])],
+      ['update', () => h(Fragment, [
+        this.isStopRequired() ?
+          p([
+            'Changing the machine type (increasing or decreasing the # of CPUs or Mem) results in an update that requires a ',
+            b(['restart']),
+            ' of your runtime. This may take a few minutes.  Would you like to proceed? ',
+            b(['(You will not lose any files.)'])
+          ]) :
+          p([
+            'Increasing the disk size or changing the number of workers (when the number of workers is >2) results in a real-time update to your runtime. ',
```

Comment:
TOL: This message is shown when there are no changes to a cluster as well. This is confusing since the language only speaks to changing/increasing things and is even more confusing when you recreate a cluster that had <2 workers (since this message specifically calls out >2 workers). I think it would be beneficial to consider adding language to cover the recreate case in here or perhaps disabling the 'Update' button if there are no changes being made to the cluster